dummy.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2016 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main_test
+
+// MainTest package makes sure these packages stay as dependencies of the root
+// module (e.g. for sake of 'bom' generation).
+import (
+	_ "go.etcd.io/etcd/client/v2"            // keep
```

Comment:
Sorry I do not follow why we need to keep these two imports here?

Reply:
'go mod tidy' would remove them without these imports. Elaborated in the comment to the file. 

Bom is generated using: 
'/home/ptab/corp/etcd/tools/mod/.gobincache/github.com/coreos/license-bill-of-materials/@v/v0.0.0-20190913234955-13baff47494e/license-bill-of-materials' '--override-file' './bill-of-materials.override.json' 'go.etcd.io/etcd/api/v3/...' 'go.etcd.io/etcd/pkg/v3/...' 'go.etcd.io/etcd/raft/v3/...' 'go.etcd.io/etcd/client/v2/...' 'go.etcd.io/etcd/client/v3/...' 'go.etcd.io/etcd/server/v3/...' 'go.etcd.io/etcd/etcdctl/v3/...' 'go.etcd.io/etcd/tests/v3/...' 'go.etcd.io/etcd/v3

That needs to be called from a 'top' of the dependencies tree.

Reply:
Done fix: 
  https://github.com/etcd-io/etcd/pull/12421